application/config/autoload.php

```diff
@@ -133,4 +133,4 @@
 |
 |	$autoload['model'] = array('first_model' => 'first');
 */
-$autoload['model'] = array('Phonebook_model');
```

Comment:
I faintly recall that this used to be required otherwise everywhere since this was required it used to load later which used to be explicitly mentioned. Does removing this no more cause that?